app/components/subject-viewer.cjsx

```diff
@@ -158,6 +158,24 @@ module.exports = createReactClass
                   <button aria-label="Play" title="Play" type="button" className="secret-button subject-tools__play" onClick={@setPlaying.bind this, true}>
                     <i className="fa fa-play fa-lg fa-fw"></i>
                   </button>}
+                <div className="subject-frame-duration">
+                  <i className="fa fa-angle-right fa-lg fa-fw subject-frame-duration--slower"></i>
+                  <input
+                    aria-valuemin="-2000"
+                    aria-valuemax="-100"
+                    aria-valuenow={@state.playFrameDuration * -1}
```

Comment:
This is duplicating the `value` attribute and can be removed.

The value of this ranfe input doesn't make a great deal of sense to me. VoiceOver, for example, announces 'range: -600', 'range: -500' etc. when I change it and I'm thinking '-500 what?'.

Can it be changed to generate a value that makes sense in terms of playback speed? I'd suggest a range from 0.25 to 1.0, for consistency with our other speed controls, and then divide the frame duration by that fraction.

Reply:
Yes! This originally made my head spin a bit, but I like your suggestion a lot. I've updated accordingly while leaving increased options at higher speeds as requested by users. I think this range makes a lot more sense. Thanks!